virtinst/domain/os.py

```diff
@@ -87,8 +88,13 @@ def _set_bootorder(self, newdevs):
         for d in newdevs:
             dev = self.bootdevs.add_new()
             dev.dev = d
-    bootdevs = XMLChildProperty(_BootDevice)
     bootorder = property(_get_bootorder, _set_bootorder)
+    bootdevs = XMLChildProperty(_BootDevice)
+    smbios_mode = XMLProperty("./smbios/@mode")
+    bootmenu_enable = XMLProperty("./bootmenu/@enable", is_yesno=True)
```

Comment:
it's a good idea to rename this, enable_bootmenu -> bootmenu_enable, but there's a usage of this in virt-manager too that needs to change as well

Reply:
Fixed & Squashed into the appropriate commit.
Sorry about that. I thought it was fine since the tests all ran through. I'll grep through the code when renaming stuff in the future.

Reply:
FWIW there are virt-manager UI tests as well: pytest --uitests. But they take over your desktop when being run, require turning on accessibility support, only work on X11, and not many people have run them besides me. They hit most of the code though. Unfortunately dynamic nature of python probably meant this wouldn't have thrown an error anyways